test/spec/modules/lotameIdSystem_spec.js

```diff
@@ -0,0 +1,144 @@
+import {lotameIdSubmodule, storage} from 'modules/lotameIdSystem.js';
```

Comment:
Does this capture all the following cases?
1) Nothing in storage to start and returns a pid, but no panorama
2) Pid in storage
     - returns pano
     - does not return pano
3) Nothing in storage to start and returns a pid and panorama (can happen if pid is in 3rd party)
4) No pid, but too early to try again
5) No pano, but too early to try again

Reply:
It doesn't really care whether or not a pid is returned but it will only ask for a new panorama if we don't have one and it's time to refresh. Now that I think about it the cookie will expire but since I'm in local storage, it will never expire the pano id

Reply:
Done